wire-library/wire-reflector/src/jvmMain/kotlin/com/squareup/wire/reflector/SchemaReflector.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2021 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.wire.reflector
+
+import com.squareup.wire.GrpcStatus
+import com.squareup.wire.schema.Location
+import com.squareup.wire.schema.Schema
+import com.squareup.wire.schema.internal.SchemaEncoder
+import grpc.reflection.v1alpha.ErrorResponse
+import grpc.reflection.v1alpha.FileDescriptorResponse
+import grpc.reflection.v1alpha.ListServiceResponse
+import grpc.reflection.v1alpha.ServerReflectionRequest
+import grpc.reflection.v1alpha.ServerReflectionResponse
+import grpc.reflection.v1alpha.ServiceResponse
+import okio.ByteString
+
+/**
+ * This converts a Wire Schema model to a protobuf DescriptorProtos model and serves that.
```

Comment:
This comment is less fitting now that the type conversion happens elsewhere.

Reply:
ok, I'll fix this in the next pr.